src/db/@types.ts

```diff
@@ -1,3 +1,24 @@
+import { AuthResult } from "express-oauth2-jwt-bearer";
+import { Request } from "express";
+
 export type ID = {
-    id: string;
-}
\ No newline at end of file
+  id: string;
+};
+
+export interface UserRequest extends Request {
+  user?: {
+    sub: string;
```

Comment:
what does sub refer to?

Reply:
Refers to the "Subject Identifier", which is a unique identifier for each user
 * obtained from the JWT issued by Auth0